frontend/challedu-app/src/App.css

```diff
@@ -87,3 +87,148 @@
     padding: 0px;
   }
 }
+
+/* Users admin table */
+
+* {
```

Comment:
By default, Bootstrap library (which has already been imported) contains this general rule that applies box-sizing border box to all elements, therefore it can be removed from this point.

Reply:
You 're right, it is not needed because it is applied by default by the bootstrap library.